app/assets/stylesheets/petitions/views/_shared.scss

```diff
@@ -230,17 +230,17 @@ input.back-page {
 .debate-video {
   position: relative;
   width: 100%;
-  padding-bottom: 56.25%; // 16:9
-  background-size: 100%;
   @include inline-block;
-  background-image: image-url("graphics/graphic_house-of-commons.jpg");
-  @include device-pixel-ratio() {
-    background-image: image-url("graphics/graphic_house-of-commons-2x.jpg");
-  }
   &:hover, &:focus {
     opacity: 0.9;
   }
 
+  img {
+    width: 100%;
+    height: 100%;
+    vertical-align: bottom;
+  }
+
```

Comment:
Without this and if the image wasn't exactly of the correct proportions, the overlay for the play button would sometimes sit slightly above the bottom of the image.

This worked ok previously because the image was a background - but on converting it to a img the browser would fit the containing div to either the image width or the image height (not both). The exact gap would also change as the page size changed.

I got Henry Turner to double check that I wasn't going crazy here.. `display:block` didn't work for this. There might be a better way. Suggestions welcome.